src/fu-release.c

```diff
@@ -772,6 +772,7 @@ fu_release_get_priority(FuRelease *self)
 static void
 fu_release_ensure_device_by_checksum(FuRelease *self, XbNode *component, XbNode *rel)
```

Comment:
I'm not sure this is the right name for this function anymore. Likewise `FU_DEVICE_PRIVATE_FLAG_MD_ONLY_CHECKSUM` feels like it's not right anymore either with this change.  Any thoughts on changing those?

Reply:
I think fu_release_ensure_device_by_checksum() is still correct; it's still doing the fu_device_ensure_from_component on any device checksum match afterall. What ideas have you got?

Reply:
`fu_release_ensure_device_by_metadata` I was thinking and maybe `FU_DEVICE_PRIVATE_FLAG_MD_ONLY_CHECKSUM_BY_PROTOCOL`?

Reply:
FU_DEVICE_PRIVATE_FLAG_MD_ONLY_CHECKSUM_BY_PROTOCOL is kinda wordy -- that said, we could do two two other things:

 * Change the InstanceIDs of the dbx2 device
 * just add some more text to the FU_DEVICE_PRIVATE_FLAG_MD_ONLY_CHECKSUM docs

Reply:
Yeah I think those are great suggestions.

Reply:
actually, you're right, the protocol check is in the wrong place entirely